### PR TITLE
add some missing include & fixed display output on sh1104

### PR DESCRIPTION
**Pull request by tuanpmt**
I got some errors when build with esphttpd

### CODE REVIEW

**Comment by Zaltora** [comment]
What problem you got with SH1106 ?   It is display good with me.  
Maybe you forget to change DISPLAY_WIDTH  to  132  because SH1106 got  132*64 buffer, even if screen is 128*64.
Need maybe update example with this details

**Comment by tuanpmt** [comment]
My display is 128x64, as the datasheet, `sh1106_go_coordinate` execute every page = 128bytes width. I can not using this library without this condition `i%dev->width == 0`

https://github.com/tuanpmt/esp-audio-player/blob/master/extras/player/eq.c#L35

**Comment by Zaltora** [comment]
This fix make sense, because currently go_coordinate is call every 16 bytes. Great fix @tuanpmt 
Just a point, i checked datasheet, the sh1106 got a 132 X 64 bits SRAM.
ref: https://www.elecrow.com/download/SH1106%20datasheet.pdf
That why in `sh1106_go_coordinate`, you got +2 offset. To center for 128X64  true screen format. I didn't know what happens if i use 128  in DISPLAY_WIDTH. If it is work well for you, maybe this part of ram is ignored. I will test that with SPI protocols.

**Comment by Zaltora** [comment]
It is work well. ^^

File: core/app_main.c [hunk 1]
```diff
@@ -21,6 +21,8 @@
 #include "esp/spi_regs.h"
 #include "esp/dport_regs.h"
 #include "esp/wdev_regs.h"
+#include "esp/wdt_regs.h"
+#include "esp/rtcmem_regs.h"
```
Comment (ourairquality) [inline core/app_main.c]: Were these includes really needed? It was not giving any warnings last time I built? Might these be included by mistake, some other local changes?

Comment (tuanpmt) [inline core/app_main.c]: the `RTCMEM_BACKUP` define inside `rtcmem_regs.h`, `WDT` inside `wdt_regs.h`. It's normal to me, except that I compiled together with esphttpd.

Comment (ourairquality) [inline core/app_main.c]: ok, thanks.

**Pull request merged**